x-pack/plugins/observability_solution/observability/public/components/custom_threshold/components/expression_row.tsx

```diff
@@ -214,12 +198,19 @@ const ThresholdElement: React.FC<{
     return threshold;
   }, [threshold, isMetricPct]);
 
+  const thresholdComparator = useCallback(() => {
+    if (!comparator) return COMPARATORS.GREATER_THAN;
+    // Check if the rule had a LEGACY_OUTSIDE_RANGE inside its params.
+    // Then, change it on-the-fly to NOT_BETWEEN
+    // @ts-ignore
+    if (comparator === LEGACY_OUTSIDE_RANGE) return COMPARATORS.NOT_BETWEEN;
```

Comment:
Legacy in the new rule 😢

Reply:
Yeah, it's the price of using another rule as a baseline.